fft/src/KokkosFFT_Plans.hpp

```diff
@@ -97,6 +97,9 @@ class Plan {
   //! The type of extents of input/output views
   using extents_type = shape_type<InViewType::rank()>;
 
+  //! Execution space
+  execSpace m_exec_space;
```

Comment:
Why do you hold on to an exec space?
When one of the API function is called presumably you take an exec space as argument and you pass the plan.  Then which exec space do you pick?  Do they have to match?  etc..

Reply:
For the moment, this internal `exec_space` is unused. I will suppress this change in this PR. 
Originally, I was inclined to use internal `exec_space` for all the execution of FFTs and Kokkos functions, but it turns out that the changes are too big to be consistent.

Reply:
Thank you for your review. I will merge this. Your reviews and/or suggestions are always appreciated!